### PR TITLE
MMDP-452 Manage Impact Type Dropdown

### PR DESCRIPTION
**Pull request by Nicanor008**
**What does this PR do?**

- Allow users to manage options in the 'Impact Type' drop down.

**description of Task to be completed?**

- Create new Impact Type Drop down
- List all mpact Type Drop downs
- Edit mpact Type Drop downs
- Delete mpact Type Drop down

**How should this be manually tested?**

* Fetch and checkout to ft-Impact-type-dropdown-MMDP-452
* Run the server `yarn run dev-server`
* On the browser, enter `http://localhost:1234/dropdowns/impact-type`

**Any background context you want to provide?**
* None

**What are the relevant Jira issues?**

[MMDP-452](https://viisaus.atlassian.net/browse/MMDP-452)

**Screenshot**
Screen Shot 2019-04-04 at 12 58 05

**Questions:**
* None

### CODE REVIEW

File: client/components/DropDowns/ImpactType/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Form, Grid } from 'semantic-ui-react';
+// import ImpactTypeDropdownForm from '../../common/Form/impactTypeDropdownForm';
```
Comment (eduhmik) [inline client/components/DropDowns/ImpactType/index.js]: Remove this chunk of commented code here

File: client/components/DropDowns/ImpactType/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import DropdownActions from '../../common/Button/DropdownActions';
+import DropdownForm from '../../common/Form/DropdownForm';
+// import getInput from './input';
```
Comment (eduhmik) [inline client/components/DropDowns/ImpactType/index.js]: Here too

**Comment by jomasim** [comment]
Add more instructions on how to test the feature on the testing section. You need to include the route to be used to access the dropdown on the browser.

File: client/components/DropDowns/ImpactType/index.js (hunk 2 above)
Comment (Nicanor008) [reply client/components/DropDowns/ImpactType/index.js]: Good catch @eduhmik , I have removed the dead code here

**Comment by Nicanor008** [comment]
> Add more instructions on how to test the feature on the testing section. You need to include the route to be used to access the dropdown on the browser.

Thanks for this feedback @jomasim

**Comment by Nicanor008** [comment]
> Please fix the linting issues. Thank you.
> Screenshot 2019-04-03 at 20 39 03

Thanks @p8ul, I have fixed the issue

**Comment by p8ul** [comment]
Good work @Nicanor008 but you missed some point when implementing impact type dropdown. Impact name is supposed to be a text field, not a select field. 
Screenshot 2019-04-04 at 11 40 45

**Comment by p8ul** [comment]
Also fix the errors on the console. 
Screenshot 2019-04-04 at 11 50 16

**Comment by Nicanor008** [comment]
> Good work @Nicanor008 but you missed some point when implementing impact type dropdown. Impact name is supposed to be a text field, not a select field.
> Screenshot 2019-04-04 at 11 40 45

@p8ul Because of impact type only direct/indirect and the best option was to use select option. 
Screen Shot 2019-04-04 at 11 49 07

File: server/__tests__/tests/api/dropdowns/impactType.spec.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        ],
+      });
+      expect(res.status).to.equal(201);
+      expect(res.body).not.to.have.property('error');
+      expect(res.body).to.have.property('message');
+      expect(res.body).to.have.property('data');
+    });
+  });
+
+  describe('GET ${route}', () => {
```
Comment (Ruiru11) [inline server/__tests__/tests/api/dropdowns/impactType.spec.js]: use string literals as opposed to quotes

Comment (Nicanor008) [reply server/__tests__/tests/api/dropdowns/impactType.spec.js]: Thanks @Ruiru11 , I have fixed

**Comment by Nicanor008** [comment]
> Good work @Nicanor008 but you missed some point when implementing impact type dropdown. Impact name is supposed to be a text field, not a select field.
> Screenshot 2019-04-04 at 11 40 45

I have made preferable changes as advised.

**Pull request merged**